index.html

```diff
@@ -2564,11 +2564,11 @@ <h2>Definition of Roles</h2>
 			</table>
 		</div>
 		<div class="role" id="definition">
-			<rdef>definition</rdef>
-			<div class="role-description">
-				<p>A definition of a term or concept. See related <rref>term</rref>.</p>
-				<p>Authors SHOULD identify the <a>element</a> being defined by giving that element a role of <rref>term</rref> and referencing it with the <pref>aria-labelledby</pref> <a>attribute</a> or by making the element with role <rref>term</rref> a descendant of the element with role <code>definition</code>.</p>
-			</div>
+      <rdef>definition</rdef>
+      <div class="role-description">
+        <p>A definition of a term or concept. See related <rref>term</rref>.</p>
+        <p>Authors SHOULD identify the <a>element</a> being defined and assign that element a role of <rref>term</rref>. In addition, authors SHOULD set <pref>aria-details</pref> on the <rref>term</rref>, pointing to the relevant <code>definition</code>.</p>
```

Comment:
```suggestion
        <p>Authors SHOULD identify the <a>element</a> being defined and assign that element a role of <rref>term</rref>. In addition, authors SHOULD set <pref>aria-details</pref> on the <rref>term</rref>, pointing to the related <code>definition</code>.</p>
```

Consider using "related" here instead of "relevant", because `aria-details` is a "Relationship attribute" which makes the `term` and `definition` "related". :)